src/components/StarRatingContainer.ts

```diff
@@ -15,10 +15,13 @@ export interface ContainerProps extends WrapperProps {
     maximumStars: number;
     onChangeMicroflow: string;
     rateAttribute: string;
+    starSize: starSize;
+    starSizeCustom: number;
     widgetColor: widgetColors;
 }
 // tslint:disable max-length-line
 export type widgetColors = "widget" | "default" | "primary" | "success" | "info" | "warning" | "danger" | "inverse" ;
```

Comment:
Type should always start with capital. `WidgetColors`
Second note, it should singular, not plural `WidgetColor` Because is is one color out of a set of possibilities,